tools/osbuild-mpp

```diff
@@ -1164,11 +1246,12 @@ class ManifestFileV2(ManifestFile):
 def main():
     parser = argparse.ArgumentParser(description="Manifest pre processor")
     parser.add_argument(
+        "--cache",
         "--dnf-cache",
         metavar="PATH",
         type=os.path.abspath,
         default=None,
-        help="Path to DNF cache-directory to use",
+        help="Path to package cache-directory to use",
```

Comment:
maybe add `dest="cachedir"` or something to make it more generic as well and then use that in line 1309.

Reply:
Makes sense, fixed!